support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.jsx

```diff
@@ -29,6 +29,7 @@ import type { PaperFulfilmentOptions } from 'helpers/productPrice/fulfilmentOpti
 import { Collection, HomeDelivery } from 'helpers/productPrice/fulfilmentOptions';
 import { paperHasDeliveryEnabled } from 'helpers/subscriptions';
 import ConsentBanner from 'components/consentBanner/consentBanner';
+import ReactDOMServer from 'react-dom/server';
```

Comment:
this is gonna bump up the filesize for the browser, could we instead return `content` from this file and import `ReactDOMServer`  from the `ssr.js` file?